src/overlays/actors/ovl_En_Zf/z_en_zf.c

```diff
@@ -2014,14 +2014,14 @@ void EnZf_UpdateDamage(EnZf* this, PlayState* play) {
                 Actor_SetColorFilter(&this->actor, 0x4000, 255, 0, 8);
 
                 if (Actor_ApplyDamage(&this->actor) == 0) {
-                    dropParams = 0x40;
+                    dropTable = DROP_TABLE_NUM(4);
                     EnZf_SetupDie(this);
 
                     if (this->actor.params == ENZF_TYPE_DINOLFOS) {
-                        dropParams = 0xE0;
+                        dropTable = DROP_TABLE_NUM(14);
                     }
 
-                    Item_DropCollectibleRandom(play, &this->actor, &this->actor.world.pos, dropParams);
+                    Item_DropCollectibleRandom(play, &this->actor, &this->actor.world.pos, dropTable);
```

Comment:
`Item_DropCollectibleRandom` takes `dropTable` as `params` so I'm not sure why `dropParams` -> `dropTable`, which one should be used? (probably params?)